### PR TITLE
add wbtc asset to axelar

### PR DESCRIPTION
**Pull request by milapsheth**

### CODE REVIEW

File: axelar/assetlist.json [hunk 1]
```diff
@@ -132,6 +132,27 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/axelar/images/weth.png"
         },
         "coingecko_id": "weth"
+      },
+      {
+        "description": "Wrapped BTC on Axelar",
```
Comment (JeremyParish69) [inline axelar/assetlist.json:137]: Expand 'BTC' to 'Bitcoin'?

File: axelar/assetlist.json [hunk 2]
```diff
@@ -132,6 +132,27 @@
...
+            "denom": "wbtc-satoshi",
+            "exponent": 0
+          },
+          {
+            "denom": "wbtc",
+            "exponent": 8
+          }
+        ],
+        "base": "wbtc-satoshi",
+        "name": "Wrapped BTC",
```
Comment (JeremyParish69) [inline axelar/assetlist.json:149]: Expand 'BTC' to 'Bitcoin'?

File: axelar/assetlist.json (hunk 1 above)
Comment (milapsheth) [inline axelar/assetlist.json:137]:
```suggestion
        "description": "Wrapped Bitcoin on Axelar",
```

File: axelar/assetlist.json (hunk 2 above)
Comment (milapsheth) [inline axelar/assetlist.json:149]:
```suggestion
        "name": "Wrapped Bitcoin",
```

**Pull request merged**